docs/source/index.rst

```diff
@@ -10,55 +10,85 @@ documented interface for SpiNNaker application developers.
 
 .. _SpiNNaker: http://apt.cs.manchester.ac.uk/projects/SpiNNaker/
 
-The following documentation aims to provide new users with a high-level
-introduction to all of the key parts of Rig and also present a formal API
-reference which is also available via Python's ``help()`` system.
+Getting started
+---------------
 
-Installation
-------------
+If you're new to Rig, here are two options for getting started: If you're
+feeling impatient and want to start playing, take a look at :ref:`some of the
+ten-line quick-start example programs <ten-lines>`. Alternatively the
+:ref:`'hello world' to circuit simulator tutorial<circuit-sim-tutorial>` gives
+a detailed introduction to building real-world SpiNNaker applications using Rig
+(still in under 400 lines of heavily commented Python).
 
 .. toctree::
-        :maxdepth: 2
+    :maxdepth: 2
+    
+    install
 
-        install
+.. toctree::
+    :maxdepth: 2
+    
+    ten_lines/index.rst
```

Comment:
I think it makes sense to have the tutorial prior to the ten-line examples, otherwise much of the content of the 10-liners is not explained well